sections/shared/Layout/AppLayout/Header/BalanceActions.tsx

```diff
@@ -32,40 +32,46 @@ const BalanceActions: FC = () => {
 
 	const sUSDBalance = synthBalances?.balancesMap?.[Synths.sUSD]?.balance ?? zeroBN;
 
-	const OPTIONS = useMemo(() => {
-		const accessiblePositions =
-			futuresPositions?.filter((position) => position.remainingMargin.gt(zeroBN)) ?? [];
+	const accessiblePositions = useMemo(
+		() => futuresPositions?.filter((position) => position.remainingMargin.gt(zeroBN)) ?? [],
```

Comment:
`futuresPositions` is empty if you visit the markets page first. If you switch to dashboard and back, it is complete. This results in the `BalanceActions` to be empty if a user visits the markets page before the dashboard.